libs/designer-ui/src/lib/tokenpicker/tokenpickersection/tokenpickeroption.tsx

```diff
@@ -181,8 +182,13 @@ export const TokenPickerOptions = ({
                     <div className="msla-token-picker-section-option-text">
                       <div className="msla-token-picker-option-inner">
                         <div className="msla-token-picker-option-title">{token.title}</div>
-                        <div className="msla-token-picker-option-description">{token.description}</div>
+                        <div className="msla-token-picker-option-description" title={token.description}>
+                          {token.description}
+                        </div>
                       </div>
+                      {token.type && token.outputInfo?.type !== TokenType.FX ? (
```

Comment:
Why do we want to show type to user?  User would know what is the parameter type they are entering on the dynamic content, these are implementation or developer logic/term right?
Also type any they wouldn't even understand. Was there any compelling reason to introduce this concept

Reply:
Why dont we have showing type in the last PR for this feature and complete others like show more and stuff. I had put some questions in the original task for this.